tests/functional/io_apis_test.c

```diff
@@ -153,13 +159,24 @@ npages_per_thread(io_test_fn_args *io_test_param, int nthreads)
 int
 splinter_io_apis_test(int argc, char *argv[])
 {
+   // Move past the 1st arg which will be the driving tag, 'io_apis_test'.
+   argc--;
+   argv++;
+
+   // If --use-shmem was provided, parse past that argument.
+   bool use_shmem = (argc >= 1) && test_using_shmem(argc, (char **)argv);
```

Comment:
Doesn't config_parse() handle that flag?

Reply:
Yes, this can be changed. Slipped my scan thru review comments previously.

Reworked this now to use `config_parse()` as the interface to parse the `--use-shmem` arg.